packages/website/ts/components/wallet/wallet.tsx

```diff
@@ -320,26 +336,23 @@ export class Wallet extends React.Component<WalletProps, WalletState> {
     private _renderEthRows(): React.ReactNode {
         const icon = <img style={{ width: ICON_DIMENSION, height: ICON_DIMENSION }} src={ETHER_ICON_PATH} />;
         const primaryText = this._renderAmount(
-            this.props.userEtherBalanceInWei,
+            this.props.userEtherBalanceInWei || new BigNumber(0),
             constants.DECIMAL_PLACES_ETH,
             constants.ETHER_SYMBOL,
+            _.isUndefined(this.props.userEtherBalanceInWei),
         );
         const etherToken = this._getEthToken();
-        const etherPrice = this.state.trackedTokenStateByAddress[etherToken.address].price;
+        const etherTokenState = this.state.trackedTokenStateByAddress[etherToken.address];
+        const etherPrice = etherTokenState.price;
         const secondaryText = this._renderValue(
-            this.props.userEtherBalanceInWei,
+            this.props.userEtherBalanceInWei || new BigNumber(0),
```

Comment:
I'm guessing using a defaultProp of new BigNumber(0) won't work because we actually want to pass `undefined` in certain cases?

Reply:
yup, we want it to show as loading